kphp_polyfills.php

```diff
@@ -326,9 +326,12 @@ function wait_synchronously($id) {
 function wait_concurrently($id) {
```

Comment:
Did you figure out what's going on in this function?
As far as I can see, it always requests undefined index at line 333, and the condition `!$waiting[$id]` is always true